modules/nw-egress-ips-static.adoc

```diff
@@ -39,10 +39,7 @@ $ oc patch netnamespace project1 --type=merge \
   -p '{"egressIPs": ["192.168.1.100"]}'
 ----
 +
-You can set `egressIPs` to two or more IP addresses on different nodes to
-provide high availability. If multiple egress IP addresses are set, pods use the
-first IP in the list for egress, but if the node hosting that IP address fails,
-pods switch to using the next IP in the list after a short delay.
+You can set `egressIPs` to two or more IP addresses on different nodes to provide high availability. If multiple egress IP addresses are set then pods use all egress IP addresses roughly equally.
```

Comment:
Maybe:

```suggestion
To provide high availability, set the `egressIPs` value to two or more IP addresses on different nodes. If multiple egress IP addresses are set, then pods use all egress IP addresses roughly equally.
```

If you're up for it, it might be helpful to provide an example of this syntax.

Reply:
Also, if you're going to strip the callouts from this many of the blocks in this assembly, you might as well do it for this module, too.